src/index.ts

```diff
@@ -0,0 +1,49 @@
+import { Tracer } from '@aws-lambda-powertools/tracer';
+import {
+  BatchGetItemCommand,
+  BatchGetItemCommandInput,
+  BatchWriteItemCommand,
+  BatchWriteItemCommandInput,
+  DeleteItemCommand,
+  DeleteItemCommandInput,
+  DynamoDBClient,
+  DynamoDBClientConfig,
+  GetItemCommand,
+  GetItemCommandInput,
+  PutItemCommand,
+  PutItemCommandInput,
+} from '@aws-sdk/client-dynamodb';
+
+export class MhDynamoDbClient {
+  private readonly client: DynamoDBClient;
+
+  constructor(config: DynamoDBClientConfig) {
+    const tracer = new Tracer();
+    this.client = tracer.captureAWSv3Client(new DynamoDBClient(config));
+  }
+
+  async getItem<T>(args: GetItemCommandInput): Promise<T> {
+    const response = await this.client.send(new GetItemCommand(args));
+    return response.Item as T;
+  }
+
+  async putItem<T>(args: PutItemCommandInput): Promise<any> {
```

Comment:
Are we using T here somewhere? If not; should we? I don't think we should return Promise<any> so either we should return whatever the client returns, or parse it as T, not sure what is preferable in a putItem scenario?